hawkular-command-gateway-api/src/main/resources/schema/A0uthMessage.schema.json

```diff
@@ -10,6 +10,30 @@
         "token":    { "type": "string" },
         "persona":  { "type": "string" }
       }
+    },
+    "recipientType": {
+      "type": "object",
+      "description": "A binary attachment metadata",
+      "enum": ["UI", "FEED"]
+    },
```

Comment:
This is not carved in stone, you may well be right, that we can live without that. My motivation for introducing it was to make the command gateway as stupid and generic as possible. I think that the gateway should be just that: transfer the messages between the web sockets and the bus, do some authentication (and perhaps authorization) but nothing more. If the messages know where they go from and to, the gateway does not need to know which makes the gateway simpler. 

Yeah, and `recipientType` allows for sending one message type to both kind of destinations, regardless from where it was sent.